sdk/core/azure_core/src/http/policies/instrumentation/request_instrumentation.rs

```diff
@@ -37,8 +41,23 @@ impl RequestInstrumentationPolicy {
     /// The tracer provided is a "fallback" tracer which is used if the `ctx` parameter
     /// to the `send` method does not contain a tracer.
     ///
-    pub fn new(tracer: Option<Arc<dyn crate::tracing::Tracer>>) -> Self {
-        Self { tracer }
+    pub fn new(
+        tracer: Option<Arc<dyn crate::tracing::Tracer>>,
+        log_options: &LoggingOptions,
```

Comment:
Nit: just for internal consistency, maybe just `logging` like `ClientOptions` has.